.github/workflows/package.yaml

```
@@ -52,6 +52,7 @@ jobs:
         shell: bash
         run: |
           gsutil rsync -d packages-selfhosted-charts/ "${CHARTS_BUCKET}"
```

Please, ignore the `index.yaml` file:
```suggestion
          gsutil -h "Cache-Control:public,max-age=3600,s-maxage=3600" -m rsync -x "^index.yaml$" -d packages-selfhosted-charts/ "${CHARTS_BUCKET}"
```